### PR TITLE
Add frequency cap example

### PR DESCRIPTION
**Pull request by jkarlin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,6 +179,44 @@ class SendReachReportOperation {
...
+    // By default, return the default url (0th index). 
+    let result = 0;
+    
+    let count = await this.sharedStorage.get(data["campaign-id"]);
+    
+    // If under cap, return the desired ad.
+    if (count < 3)
+      result = 1;
+    
+    this.sharedStorage.set(data["campaigh-id"], count + 1);
```
[the remark this answers is not in the document]
Comment (jkarlin) [reply README.md]: Thanks! Done and done.

File: README.md [hunk 2]
```diff
@@ -181,6 +179,45 @@ class SendReachReportOperation {
...
+  async function run(data, urls) {
+    // By default, return the default url (0th index). 
+    let result = 0;
+    
+    let count = await this.sharedStorage.get(data["campaign-id"]);
+    
+    // If under cap, return the desired ad.
+    if (count < 3) {
+      result = 1;
+      this.sharedStorage.set(data["campaign-id"], count + 1);
```
Comment (pythagoraskitty) [inline README.md:214]: Won't this do concatenation? Do you need to convert `count` to an integer here first?

Comment (pythagoraskitty) [reply README.md:214]: Cool, ok.  I wasn't sure, was just looking at https://www.tutorialspoint.com/how-to-add-a-number-and-a-string-in-javascript

Comment (pythagoraskitty) [reply README.md:214]: But LGTM otherwise.

Comment (jkarlin) [reply README.md:214]: Wait, you're right! I could swear I tried this out in console. 

Good catch, updating.

Comment (jkarlin) [reply README.md]: Done, PTAL

**Review by pythagoraskitty — APPROVED** [review]

**Pull request merged**